packages/identity-service/sequelize/migrations/20240610205258-add-address-to-auth-table.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('Authentications', 'walletAddress', {
+        type: Sequelize.STRING,
+        allowNull: true
```

Comment:
do we want to add a unique constraint in here too?

Reply:
we don't want this as it's how email changing works

Reply:
Yeah idt it can be unique and nullable